gulpfile.js

```diff
@@ -9,9 +9,9 @@ require('./tasks/gulp/sass')
 require('./tasks/gulp/serve')
 require('./tasks/gulp/jest')
 
-gulp.task('watch', ['build:watch', 'scss:watch', 'copy-assets:watch', 'jest:unit:watch'])
+gulp.task('watch', ['build:watch', 'scss:watch', 'copy-assets:watch'])
 
-gulp.task('build', ['clean', 'compile', 'scss:compile', 'copy-assets'])
+gulp.task('build', ['lint', 'clean', 'compile', 'scss:compile', 'copy-assets'])
```

Comment:
Hmm, I'm not convinced by this.  I'd personally run: `npm test && npm run build`, in a number of projects I've actually found it a problem when you can't build with failing stuff - mostly for local testing.

Reply:
👍